### DISCUSSION TITLE
how to let hmtl5、vue.js、nginx  integrated spring cloud gateway

### DISCUSSION THREAD

**Discussion by czxin788** [apache/skywalking#8782] [Q&A]
I know skywaling can integrated apisix or kong gateway，but how to let hmtl5、vue.js、nginx  integrated spring cloud gateway

**Comment by wu-sheng** [comment]
What do you mean to integrate? Do you mean monitoring browser applications?

**Reply by czxin788** [reply]
Yes, I connect nginx Lua to skywalking, but only display the tracking information of static pages such as HTML, CSS and JS.
But that's not what I want.
How to display the complete trace metrics between the front end(such as html、js、css) and the back end（such as java）

**Reply by wu-sheng** [reply]
OpenResty to Java has been there for years. I am not sure what are you missing.

On the other hand, JS client side agent is there for years too, https://skywalking.apache.org/docs/#ClientJavaScript.

Read docs before you submit any questions.
